app/tests/conftest.py

```
@@ -81,6 +125,34 @@ async def _reset_redis_singleton():
+            if name in REGISTRY._children:
+                REGISTRY._children.pop(name, None)
+        except (KeyError, AttributeError):
+            pass
+    
+    yield
+    
+    # Clean up again after test
+    collectors_to_remove = list(REGISTRY._names_to_collectors.keys())
+    for name in collectors_to_remove:
+        try:
+            collector = REGISTRY._names_to_collectors.pop(name)
```

Ruff (0.15.0)

[error] Local variable `collector` is assigned to but never used

Remove assignment to unused variable `collector`

(F841)